{{cookiecutter.project_slug}}/config/settings/base.py

```diff
@@ -76,6 +76,11 @@
     "allauth.socialaccount",
     "rest_framework",
 ]
+# http://docs.celeryproject.org/en/latest/userguide/periodic-tasks.html#using-custom-scheduler-classes
+{% if cookiecutter.use_celery == 'y' -%}
+THIRD_PARTY_APPS += ["django_celery_beat"]
+{%- endif %}
```

Comment:
Do you know why it failed when you added it to the list above directly?

Reply:
Not sure. I think the file in the generated project didn't have the right formatting.

Reply:
Yea, our error reporting is really bad on CI at the moment, I will try to improve this... It has to do with the position of the dash in the Jinja tags. You wrote:

```
    "rest_framework",
{% if cookiecutter.use_celery == 'y' -%}
    "django_celery_beat",
{%- endif %}
]
```

Which is compiling to:

```
    "rest_framework",
"django_celery_beat",
]
```

Instead of:

```
    "rest_framework",
{%- if cookiecutter.use_celery == 'y' %}
    "django_celery_beat",
{%- endif %}
]
```

Which is compiling to, and makes the linters happy:

```
    "rest_framework",
    "django_celery_beat",
]
```

Reply:
Gotcha! I was wondering what the dashes do. Fixed in the new commit